census/tests/test_google_publish.py

```diff
@@ -0,0 +1,121 @@
+from datetime import datetime
+import logging
+from unittest import mock
+
+from django.core.management import call_command
+from django.test import TestCase
+from pytz import timezone
+import recurrence
+
+from census import constants, google_calendar
+from census.models import Event, GoogleEvent
+import census.tests.factory as factory
+
+
+#TODO should this live somewhere else?
+def setUpModule():
+    logging.disable(logging.WARNING)
+
+
+def tearDownModule():
+    logging.disable(logging.NOTSET)
+
+
+class GooglePublishCommandTest(TestCase):
+    @mock.patch('census.google_calendar.google_publish_event')
+    def test_approved_events(self, mock_google_publish_event):
+        """Given an approved event, the event should be published"""
+
+        approved_event = factory.event(approval_status=constants.EventApprovalStatus.APPROVED)
+        approved_event.save()
+
+        call_command('google_publish')
+
+        mock_google_publish_event.assert_called_with(approved_event)
+
+    @mock.patch('census.google_calendar.google_publish_event')
+    def test_pending_events(self, mock_google_publish_event):
+        """Given pending event, the event should not be published"""
+
+        pending_event = factory.event(approval_status=constants.EventApprovalStatus.PENDING)
+        pending_event.save()
+
+        call_command('google_publish')
+
+        mock_google_publish_event.assert_not_called()
```

Comment:
I don't see where in google_publish.py we only publish APPROVED events.  The code is 
 for event in Event.objects.all(): 
And there is no check for EventApprovalStatus there or in google_publish_event.  Why does this test work then?